src/main/java/seedu/address/logic/parser/AddTaskParser.java

```diff
@@ -13,10 +19,15 @@
 
     private final AddFloatingTaskParser addFloatingTaskParser;
 
+    private final Command incorrectCommand =
+            new IncorrectCommand(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+            AddTaskCommand.MESSAGE_USAGE
+            + AddDeadlineCommand.MESSAGE_USAGE
+            + AddEventCommand.MESSAGE_USAGE
+            + AddFloatingTaskCommand.MESSAGE_USAGE));
+
```

Comment:
If I had infinite time and energy, I would get rid of the annoying `IncorrectCommand` / `CommandResult` thing and replace them with exceptions.